platform/config-core/src/main/java/org/motechproject/config/core/service/CoreConfigurationService.java

```diff
@@ -74,6 +74,12 @@
     ConfigLocation getConfigLocation();
 
     /**
+     * Returns the Motech location where all Motech files are present.
+     * @return configLocation.
+     */
+    ConfigLocation getMotechLocation();
```

Comment:
Is this the motech directory? We shouldn't use the ConfigLocation object for this